pt/scripts/settlement.py

```diff
@@ -0,0 +1,32 @@
+import json
+import random
+from datetime import datetime
+
+
+def get_settlement():
+    achievement_list = list(range(0, 105, 5))
+    transactions = [
+        {
+            "seller": "Carlab_BEMS",
+            "buyer": "SGESC_C_BEMS",
+            "achievement": random.choice(achievement_list) / 100
+        },
+        {
+            "seller": "SGESC_D_BEMS",
+            "buyer": "SGESC_C_BEMS",
+            "achievement": random.choice(achievement_list) / 100
+        },
+        {
+            "seller": "SGESC_C_BEMS",
+            "buyer": "ABRI_BEMS",
+            "achievement": random.choice(achievement_list) / 100
+        }
+    ]
+    now = datetime.now()
+    data = {
+        "date": now.strftime("%Y/%m/%d"),
+        "time": now.strftime("%H:%M"),
+        "transactions": transactions
+    }
+
+    return json.dumps(data)
```

Comment:
This function is solely for generating test settlement for front-end use case. If we start working on true settlement calculation, we can do it in this function too, in the future

Reply:
OK, maybe you can write this purpose on top of this file

How about renaming this file to test_settlement.py?

Reply:
fixed, changed the file name to test_settlement.py